examples/placeholder/linux/include/operational-state-delegate-impl.h

```diff
@@ -0,0 +1,133 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app-common/zap-generated/cluster-objects.h>
+#include <app/clusters/operational-state-server/operational-state-delegate.h>
+#include <app/util/af-enums.h>
+#include <protocols/interaction_model/StatusCode.h>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace OperationalState {
+
+/*
+ * An example state (not in spec) to present a device that is unable to honour the Pause/Resume/Start/Stop command
+ */
+enum class ManufactureOperationalStateEnum : uint8_t
+{
+    kChildSafetyLock = 0x80,
```

Comment:
All the same comments as for the examples in all-clusters-app apply here.

I strongly suggest not adding this to the placeholder app at all until the all-clusters-app implementation is in good shape.